src/main/java/racingcar_tdd/Application.java

```diff
@@ -0,0 +1,57 @@
+package racingcar_tdd;
+
+import racingcar_tdd.domain.Car;
+import racingcar_tdd.domain.Cars;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Application {
+
+    public static void main(String args[]) {
+
+        Scanner input = new Scanner(System.in);
+
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        List<String> carnamelist = Arrays.asList(input.next().split(","));
```

Comment:
변수명에 컬렉션 타입을 넣는 것은 지양하는 것이 좋습니다.
구현이 달라져 컬렉션이 변경될 때 이름도 함께 변경이 필요하다는 단점이 있습니다.
```suggestion
        List<String> carNames = Arrays.asList(input.next().split(","));
```